proposals/partial-type-inference-v2.md

```diff
@@ -160,7 +160,7 @@ The initial set of candidate methods for is changed by adding new condition.
 - If `F` is generic and `M` has no type argument list, `F` is a candidate when:
   - Type inference (§12.6.3) succeeds, inferring a list of type arguments for the call, and
   - Once the inferred type arguments are substituted for the corresponding method type parameters, all constructed types in the parameter list of `F` satisfy their constraints ([§8.4.5](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/types.md#845-satisfying-constraints)), and the parameter list of `F` is applicable with respect to `A` (§12.6.4.2)
-- **If `F` is generic and `M` has a type argument list containing at least one *inferred_type* or *partial_inferred_type*, `F` is a candidate when:**
+- **If `F` is generic and `M` is *partial_inferred*, `F` is a candidate when:**
   - **Method type inference (See the [Type Inference] section) succeeds, inferring the type arguments list for the call, and**
```

Comment:
Was Type Inference meant to be a link here?

Reply:
https://github.com/dotnet/csharplang/pull/7582/files#diff-79921a867e573503f54d5a13d465b08a27cf745e8ed47b914438e95fd17a4edcR139

Its just a name for these types of expressions (`M<_, List<_>>(...)`)